doc/classes/AnimationTreePlayer.xml

```diff
@@ -610,6 +619,16 @@
 	</methods>
 	<members>
 		<member name="playback_process_mode" type="int" setter="set_animation_process_mode" getter="get_animation_process_mode" enum="AnimationTreePlayer.AnimationProcessMode">
+			The process notification in which to update animations. Default value: [enum ANIMATION_PROCESS_IDLE].
+		</member>
+		<member name="master_player" type="NodePath" setter="set_master_player" getter="get_master_player">
+			The [AnimationPlayer] node responsible for controlling this tree's animations.
+		</member>
+		<member name="base_path" type="NodePath" setter="set_base_path" getter="get_base_path">
+			The node from which to relatively access other nodes. Default value: [code]".."[/code].
```

Comment:
Actually in case of AnimationTreePlayer as far as I was using it, the only place where I can think where this has any meaning for the user are the filter paths from `animation_node_set_filter_path` or ui (the same thing): https://i.imgur.com/5OtQODp.png